src/chart/parameter/component/NodePropertyParameterSelect.tsx

```diff
@@ -11,10 +11,23 @@ const NodePropertyParameterSelectComponent = (props: ParameterSelectProps) => {
       ? props.settings.defaultValue
       : '';
 
+  const getInitialValue = (value, multi) => {
+    if (value && Array.isArray(value)) {
```

Comment:
I think a potential problem is introduced because Array.isArray() returns `true` for strings

Reply:
Liar